Misc/NEWS.d/next/Core and Builtins/2021-12-13-17-12-16.bpo-44525.4-FiSf.rst

```diff
@@ -0,0 +1,8 @@
+Replace the four call bytecode instructions which one pre-call instruction
+and two call instructions.
+
+Removes ``CALL_FUNCTION``, ``CALL_FUNCTION_KW``, ``CALL_METHOD`` and
+``CALL_METHOD_KW``.
+
+Adds ``CALL_NO_KW`` and ``CALL_KW`` call instructions, and
+``PRECALL_METHOD`` prefix for pairing with ``LOAD_METHOD``.
```

Comment:
We've started keeping another log of bytecode changes over in [What's New](https://github.com/python/cpython/blob/main/Doc/whatsnew/3.11.rst#cpython-bytecode-changes). That section should probably be updated with these changes (especially since `CALL_METHOD` and `CALL_METHOD_KW` are mentioned there already).